src/resolver.cpp

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2023 The Foundry Visionmongers Ltd
+
+#include "resolver.h"
+
+#include "pxr/usd/ar/assetInfo.h"
+#include "pxr/usd/ar/defaultResolver.h"
+#include "pxr/usd/ar/defineResolver.h"
+#include "pxr/base/tf/debug.h"
+#include "pxr/base/tf/diagnostic.h"
+
+PXR_NAMESPACE_USING_DIRECTIVE
+PXR_NAMESPACE_OPEN_SCOPE
+
+AR_DEFINE_RESOLVER(UsdOpenAssetIOResolver, ArResolver);
+
+TF_DEBUG_CODES(OAIO_RESOLVER);
```

Comment:
[fix] We can't use the OAIO acronym, see https://github.com/OpenAssetIO/OpenAssetIO/issues/153

Use `OPENASSETIO_RESOLVER` or similar instead. Is it also worth making sure we disambiguate "resolver" (ie that were in USD) given than OpenAssetIO is also hosting a resolver 🙃

Reply:
[e48a3c8](https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/3/commits/e48a3c80eeea2797a7eccf18286aead15cceb6ac)

Changed to `OPENASSETIO_RESOLVER` as suggested. Also in this fixup expanded the debug logging to be formatted better, see https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/3#issuecomment-1387046531
It's just done in this fixup because it's all on the same lines and i'm trying to sidestep annoying rebase problems.

Reply:
Perfect, thanks!